src/proto/message_translation/tbots_protobuf.h

```diff
@@ -189,6 +214,21 @@ BallState createBallState(const TbotsProto::BallState ball_state);
 std::unique_ptr<TbotsProto::PassVisualization> createPassVisualization(
     const std::vector<PassWithRating>& passes_with_rating);
 
+/**
+ * Returns an attacker visualization
+ *
+ * @param pass An optional pass to visualize
+ * @param pass_committed Whether we are committed to taking the pass
+ * @param shot An optional shot to visualize
+ * @param balls_position The current position of the ball used to visualize the shot
+ * @param chip_target An optional target that we are chipping to
+ *
+ * @return The unique_ptr to an AttackerVisualization proto
+ */
+std::unique_ptr<TbotsProto::AttackerVisualization> createAttackerVisualization(
+    const std::optional<Pass>& pass, bool pass_committed, const std::optional<Shot>& shot,
+    const std::optional<Point>& balls_position, const std::optional<Point>& chip_target);
+
```

Comment:
maybe consider moving this to `AttackerTactic`

Reply:
Good idea! I've moved it to `AttackerTactic`